lib/schemes/oauth2.js

```diff
@@ -5,7 +5,9 @@ const isHttps = process.server ? require('is-https') : null
 const DEFAULTS = {
   token_type: 'Bearer',
   response_type: 'token',
-  tokenName: 'Authorization'
+  tokenName: 'Authorization',
+  token_key: 'access_token',
+  refresh_token_key: 'refresh_token'
```

Comment:
Yes. You can use both of these variables to change the parameter the auth module looks for to get keys. FWIW, I've never seen a use for specifying an alternate refresh token key, but it's in the original auth module functionality.

Reply:
Oops. Sorry, misunderstood the question. The underscore was already being used in the module so I did not change to prevent breakage. However, if you want to change it throughout the code, it should work fine.

Reply:
I also think I should use `tokenKey`, but that will cause a break

Reply:
Perhaps this will create another PR with this integration and letting users know that these options have changed

Reply:
Example:
```js
if (this.options.token_key && !this.options.tokenKey) {
   this.options.tokenKey = this.options.token_key;
   consola.warn('The options `token_key` has been renamed to `tokenKey`.');
}
```